include/freetds/bool.h

```diff
@@ -20,6 +20,8 @@
 #ifndef freetds_bool_h_
 #define freetds_bool_h_
 
+#include <config.h>
+
```

Comment:
Including `config.h` is pretty standard in the source files, not in headers.

Reply:
That's fair.  I would like for it to land before any declarations, but another look appears to indicate that adjusting `src/ctlib/unittest/common.h` to include `config.h` ahead of `ctpublic.h` should suffice at this point.